app/src/main/res/values/styles.xml

```diff
@@ -62,5 +62,8 @@
     <style name="MaterialDatePickerTheme" parent="Theme.AppCompat.Light.Dialog">
         <item name="colorAccent">@color/colorAccent</item>
     </style>
+    <style name="Toolbar.Titile" parent="@style/TextAppearance.AppCompat.Title">
```

Comment:
Fix the typo `Titile` and don't use hardcode size, use any existing size or if any existing size doesn't exist, create new size in dimens.xml

Reply:
@miPlodder Done.